routes/api.php

```diff
@@ -14,8 +13,9 @@
 |
 */
 
-Route::middleware('auth:api')->get('/user', function (Request $request) {
-    return $request->user();
-});
+Route::post('register', 'AuthController@register')->name('register');
```

Comment:
Please wrap all these route in a group and add prefix 'v1' so the url will be api/v1/endpoint
this is a convention to keep your controller in versions in case we have second release.
Also Nourhan did that it her task.

Reply:
I actually just asked Nourhan to remove it.

Reply:
I believe it should be exists. What do you think

Reply:
I think it is really not necessary. There won't be a V2. Let's just hope our current APIs ship safely.

Reply:
@RanaEmad That will not take much effort to ignore it. It is very simple and we just trying to follow the commonly-used convention.